lms/djangoapps/support/static/support/js/views/enrollment_modal.js

```diff
@@ -0,0 +1,76 @@
+;(function (define) {
+    'use strict';
+
+    define([
+        'backbone',
+        'underscore',
+        'gettext',
+        'text!support/templates/enrollment-modal.underscore'
+    ], function (Backbone, _, gettext, modalTemplate) {
+        var EnrollmentModal = Backbone.View.extend({
+            events: {
+                'click .enrollment-change-submit': 'submitEnrollmentChange',
+                'click .enrollment-change-cancel': 'cancel'
+            },
+
+            initialize: function (options) {
+                this.enrollment = options.enrollment;
+                this.modes = options.modes;
+                this.reasons = options.reasons;
+                this.template = modalTemplate;
+            },
+
+            render: function () {
+                this.$el.html(_.template(this.template, {
+                    enrollment: this.enrollment,
+                    modes: this.modes,
+                    reasons: this.reasons,
+                }));
+                return this;
+            },
+
+            show: function () {
+                this.$el.removeClass('is-hidden').addClass('is-shown');
+                this.render();
+            },
+
+            hide: function () {
+                this.$el.removeClass('is-shown').addClass('is-hidden');
+                this.render();
+            },
+
+            showErrors: function (errorMessage) {
+                this.$('.enrollment-change-errors').text(errorMessage).show();
```

Comment:
Nitpick: jQuery's .hide and .show functions have a weird performance hit due to an edge case they cover. It would be cleaner to add/remove classes.